lib/active_model_serializers/cancan/has_many.rb

```diff
@@ -0,0 +1,14 @@
+module ActiveModel
+  class Serializer
+    module CanCan
+      module HasMany
+        def serialize
+          return super unless authorize?
+          associated_object.select {|item| find_serializable(item).can?(:read, item) }.map do |item|
```

Comment:
not sure if this is the fastest way of doing this
can you share more data here

I am worried about really large associations being loaded in memory, where cancancan will do a load, and only use .can?(read) IF authorization is a block, where if its not it will rely on sql

Reply:
this is the code we're already using from the other branch  [`df-hack_v0.0.5_for_R52`](https://github.com/glebtv/active_model_serializers_cancancan/compare/master...CareMessagePlatform:df-hack_v0.0.5_for_R52?expand=1)

Any ideas to improve this? I could use some help.